docs/products/clickhouse/howto/create-dictionary.md

```diff
@@ -82,13 +79,11 @@ with two exceptions,`ssd_cache` and `complex_key_ssd_cache`, which are not suppo
 - HTTP(s)
 - Upstream/remote ClickHouse
 - Aiven for ClickHouse
-
-<!--
 - Upstream/remote MySQL®
 - Aiven for MySQL
 - Upstream/remote PostgreSQL®
 - Aiven for PostgreSQL
--->
```

Comment:
Change `Upstream/remote` to just `Remote`

Reply:
maybe external would be better

Reply:
Hesitated on external, will it be easily perceived easily as "external to the platform" vs. remote which is clearly "outside the node"?